drivers/bridge/bridge.go

```diff
@@ -647,7 +646,7 @@ func (d *driver) createNetwork(config *networkConfiguration) error {
 		id:         config.ID,
 		endpoints:  make(map[string]*bridgeEndpoint),
 		config:     config,
-		portMapper: portmapper.New(d.config.UserlandProxyPath),
+		portMapper: portmapper.New(portmapper.WithUserlandProxy(d.config.EnableUserlandProxy, d.config.UserlandProxyPath)),
```

Comment:
I know it was already this way. Just wondering if it makes sense to pass a proxyPath when `d.config.UserlandProxyPath==false`.
Given we are refarctoring this now, should the create option only contain the path and be passed only if `d.config.UserlandProxyPath==true`